test/functionalTest/cases/1582_alarmManager/alarm_manager_bad_input.test

```diff
@@ -0,0 +1,169 @@
+# 03. GETT /v2/entities - bad verb
+# 04. POST /v2/entities without payload
+# 05. GET /v2/entities to reset Bad Input
+# 06. One more error, just to show how it looks in the log file ...
+# 07. grep in log-file
+# 
+
+echo "01. PUT /v2/noService"
+echo "====================="
+orionCurl --url /v2/noService --json
+echo
+echo
+
+
+echo "02. POST /v2/entities with invalid JSON"
+echo "======================================="
+payload='abc'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "03. GETT /v2/entities - bad verb"
+echo "================================"
+orionCurl --url /v2/entities -X GETT --json
+echo
+echo
+
+
+echo "04. POST /v2/entities without payload"
+echo "====================================="
+orionCurl --url /v2/entities --json -X POST
+echo
+echo
+
+
+echo "05. GET /v2/entities to reset Bad Input"
+echo "======================================="
+orionCurl --url /v2/entities --json
+echo
+echo
+
+echo "06. One more error, just to show how it looks in the log file ..."
+echo "================================================================="
+orionCurl --url /v2/entities --json -X PROUST
+echo
+echo
+
+echo "07. grep in log-file"
+echo "===================="
+egrep 'Raising alarm|Releasing alarm' /tmp/contextBroker.log | awk -F'|' '{ print $9 }' | awk -F']:' '{ print $2 }'
+echo
+echo
+
+
+--REGEXPECT--
+01. PUT /v2/noService
+=====================
+HTTP/1.1 400 Bad Request
+Content-Length: 56
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "service not found",
+    "error": "BadRequest"
+}
+
+
+02. POST /v2/entities with invalid JSON
+=======================================
+HTTP/1.1 400 Bad Request
+Content-Length: 75
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "Errors found in incoming JSON buffer",
+    "error": "ParseError"
+}
+
+
+03. GETT /v2/entities - bad verb
+================================
+HTTP/1.1 405 Method Not Allowed
+Content-Length: 0
+Allow: POST, GET
+Date: REGEX(.*)
+
+
+
+04. POST /v2/entities without payload
+=====================================
+HTTP/1.1 411 Length Required
+Content-Length: 91
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "Zero/No Content-Length in PUT/POST/PATCH request",
+    "error": "LengthRequired"
+}
+
+
+05. GET /v2/entities to reset Bad Input
+=======================================
+HTTP/1.1 200 OK
+Content-Length: 2
+Content-Type: application/json
+Date: REGEX(.*)
+
+[]
+
+
+06. One more error, just to show how it looks in the log file ...
+=================================================================
+HTTP/1.1 405 Method Not Allowed
+Content-Length: 0
+Allow: POST, GET
+Date: REGEX(.*)
+
+
+
+07. grep in log-file
+====================
+ Raising alarm BadInput 0.0.0.0: service '/v2/noService' not found
+ Releasing alarm BadInput 0.0.0.0
+ Raising alarm BadInput 0.0.0.0: cannot find default service path for: (PROUST /v2/entities) - BAD VERB?
```

Comment:
Did you see the traces in the log file?

Instead of complaining about BadVerb, we have the "cannot find default service path".
It seems we do that check before checking for bad verb (normal).
I added the text "Bad Verb?" to 'fix' this problem ...